src/game/client/components/menus.h

```diff
@@ -239,8 +239,8 @@ class CMenus : public CComponent
 	// Listbox : found in menus_listbox.cpp
 	struct CListboxItem
 	{
-		int m_Visible;
-		int m_Selected;
+		bool m_Visible;
+		bool m_Selected;
```

Comment:
Minor nitpick, and I'm not very knowledgeable about this, but `bool`s in structs that contain no pointers is something I always avoid, because they can leave some empty padding, and that causes issues whenever I use algorithms that implicitly do memcmps on that. Is that a silly concern?

Reply:
Hmm, you may be right. I do mostly Java and am not used to dealing with low level. But you are probably not going to use any algorithms on a CListBoxItem anyway. It's just a container object that is destroyed almost instantly.

Besides I'll probably rework that class some more anyway, because m_Selected appears to be unused anyway.